agent/consul/stream/event.go

```diff
@@ -0,0 +1,33 @@
+package stream
+
+type Topic int32
+
+// TODO: remove underscores
+// TODO: type string instead of int?
+const (
+	Topic_ServiceHealth        Topic = 0
+	Topic_ServiceHealthConnect Topic = 1
+	Topic_ACLTokens            Topic = 2
+	Topic_ACLPolicies          Topic = 3
+	Topic_ACLRoles             Topic = 4
```

Comment:
I've been wondering about the relationship between topics and memDB table names. Do we expect them to always be 1:1, or M:1 ? 

One of the disadvantages to the approach of using local and `structs` types is that we will need to map these topic indexes when we create the gRPC events. If there is a consistent mapping of topics->tables we could resolve that issue by using table names as the topic internally, and having only a single mapping between them.

Reply:
After looking at this a bit more, I think it wont be a problem. The topics can be enumerated anywhere, `stream` doesn't need to know the list of topics.